theurgy.lic

```diff
@@ -744,11 +744,24 @@ class TheurgyActions
 
   def wine
     walk_to_altar_or_prayer_mat
-    DRCI.get_item_safe('wine', @theurgy_supply_container)
-    if @prayer_mat
-      fix_standing
-      bput("kneel #{@prayer_mat}", 'You humbly kneel')
+    if DRCI.get_item?("wine", @theurgy_supply_container)
+      if @prayer_mat
```

Comment:
The old code would try to use a prayer mat if it was defined, but continue if not.

The new code returns if no prayer mat is defined. Is that intended?